mass_mailing_partner/models/res_partner.py

```diff
@@ -35,16 +35,22 @@ def _check_email_mass_mailing_contacts(self):
                 _("This partner '%s' is subscribed to one or more "
                   "mailing lists. Email must be assigned." % self.name))
 
-    @api.one
+    @api.multi
     @api.depends('mass_mailing_contact_ids',
                  'mass_mailing_contact_ids.opt_out')
     def _compute_mass_mailing_contacts_count(self):
-        self.mass_mailing_contacts_count = len(self.mass_mailing_contact_ids)
+        for partner in self:
+            partner.mass_mailing_contacts_count = len(
+                self.env['mail.mass_mailing.contact'].search_read(
```

Comment:
Better use [`search_count()`](https://github.com/odoo/odoo/blob/1dae7ea1d660dca32e7261c5c0ed710e5541c641/openerp/models.py#L1638-L1647), which performs a DB-side count, quite faster too.

Reply:
Yes, indeed, I didn't think on it.